conda.recipe/meta.yaml

```
@@ -0,0 +1,52 @@
+package:
+  name: pyqstem
+  # this is nice, but only works once you have an initial annotated git tag
+  # version: {{ GIT_DESCRIBE_TAG }}.{{ GIT_BUILD_STR }}
+  version:  1.0
+
+source:
+  path: ..
+  # git_url: https://github.com/jacobjma/PyQSTEM
+  # git_tag:  (commit id or tag of release to build, defaults to master)
+
+build:
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - cython
```

This makes the compilers easier to deal with.  You can read more at https://docs.conda.io/projects/conda-build/en/latest/resources/compiler-tools.html

```suggestion
    - {{ compiler("c") }}
    - {{ compiler("cxx") }}
    - cython
```

---

This makes it possible to compile in the normal Anaconda command prompt window.  Unfortunately, it causes linkage to fail (both in Anaconda and Visual Studio windows) as it fails to find fftw3.lib.  I am confused.